person.rb

```diff
@@ -0,0 +1,26 @@
+# Person class
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    if is_of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] Kindly take note of your method name. You have `of_age?` but you calling `is_of_age?` here. Kindly fix that.

- [ ] You also put this method under the `private` method but it should be public. Kindly fix that.

- **[OPTIONAL]** We can make this method more brief.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```

Reply:
@DammyShittu  thanks for the comment  and  have  fixed , but for the on your second suggestion  which is` " You also put this method under the private method but it should be public. Kindly fix that. "` on the instuction it said  private method  here it is  `Private method is_of_age? that returns true if @age is greater or equal to 18 and false otherwise`. can you explain for me further?